src/makeRobot.js

```diff
@@ -38,7 +38,63 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
```

Comment:
you could've return the whole object right away without assigning into variable,
although it's totally fine this way too
```suggestion
  return {
```